app/src/main/java/com/antosik/benchproject/app/main/view/MainActivity.kt

```diff
@@ -26,5 +26,14 @@ class MainActivity : AppCompatActivity() {
         val navController = navHostFragment.navController
         val bottomNavigationView = binding.activityMainBottomNavigation
         setupWithNavController(bottomNavigationView, navController)
+
+        navController.addOnDestinationChangedListener { _, destination, _ ->
```

Comment:
Create an issue to later extract it to a specific class.

Reply:
Done #122